File: Python/08-Encoded-Result.py
def prod(s): # Function to find the digit product from a string
    p = 1
    for i in s:
        if i.isdigit():
            p *= int(i)
    return p

for _ in range(int(input())): # Loop for testcases
    print(prod(input())) # Printing the product

File: C++/08-Encoded-Result.cpp
#include <iostream>
using namespace std;
#include <stdio.h>
int main()
{
    int n;
    cin>>n;
    while(n>0){
        char string[10000];
        int count;
        long long int prod = 1;
        
        cin>>string;        // intaking string input
        
        for (count = 0; string[count]!='\0'; count++)         //traversing the string
        {
            if ((string[count] >= '0') && (string[count] <= '9'))     //checking for digit
            {
                prod *= (string[count] - '0');          //multiplying the digits
            }
        }
        cout<<prod<<endl;
        n--;
    }
    return 0;
}
